docs/src/components/ConfigurationDrawer.tsx

```diff
@@ -130,6 +139,21 @@ export const ConfigurationDrawer = ({
 
       <VuiSpacer size="m" />
 
+      <VuiFormGroup label="Response Language" labelFor="responseLanguage">
+        <VuiSelect
+          value={language}
+          onChange={(evt) => {
+            onUpdateLanguage(evt.target.value);
+          }}
+          options={SUMMARY_LANGUAGES.filter((lang) => lang !== "auto").map((lang) => ({
```

Comment:
Much like the previous experience we had with it, "auto" still isn't quite working correctly- no matter the input language, the output was always German. (This was the reason we hardcoded to "eng").

So instead of exposing something that isn't working correctly at the moment (and is also the top option in the dropdown), I filtered it out for now, as realistically, our users won't get any benefit from it.